mycroft/enclosure/gui.py

```diff
@@ -53,6 +61,42 @@ def setup_default_handlers(self):
         msg_type = self.build_message_type('set')
         self.skill.add_event(msg_type, self.gui_set)
 
+        # TODO can we rename this namespace to mycroft.playback.XXX ?
+        self.skill.add_event('mycroft.audio.service.pause',
+                             self.__handle_gui_pause)
+        self.skill.add_event('mycroft.audio.service.resume',
+                             self.__handle_gui_resume)
+        self.skill.add_event('mycroft.audio.service.stop',
+                             self.__handle_gui_stop)
+        self.skill.add_event('mycroft.audio.service.track_info',
+                             self.__handle_gui_track_info)
+        self.skill.add_event('mycroft.audio.queue_end',
+                             self.__handle_gui_stop)
+        self.skill.gui.register_handler('video.media.playback.ended',
```

Comment:
Was just giving this a run on a Mark II and this message doesn't seem to get triggered. Given it overrides idle this leaves the screen black until you trigger some other GUI event. I'm guessing this isn't happening on other platforms?

Also wondering if we should make this consistent with the other messages, perhaps `mycroft.video.playback.ended`

Reply:
is setup_default_handlers() even called anywhere, I don't see the handle even registering alone with this PR, maybe a missing piece in some other PR ?

Reply:
Nevermind, it was my bad, `setup_default_handlers()` is called. I hadn't pulled in the latest changes on the device.